### PR TITLE
Use build.info not file-wide ifndef

### PR DESCRIPTION
**Pull request by richsalz**
If configured with no-cms, handle it in build.info like the other options.
I guess I missed doing this file in PR #11250

### CODE REVIEW

File: apps/cms.c [hunk 1]
```diff
@@ -276,6 +274,7 @@ int cms_main(int argc, char **argv)
     CMS_ReceiptRequest *rr = NULL;
     ENGINE *e = NULL;
     EVP_PKEY *key = NULL;
+    char *wrapname = NULL;
```
Comment (slontis) [inline apps/cms.c:277]: The changes from here down seem not related to this PR..

Comment (richsalz) [reply apps/cms.c]: Sigh.  I tried to split this off from #14182 and did not do a good job.   fixup commit pushed.

File: apps/cms.c [hunk 2]
```diff
@@ -972,7 +970,7 @@ int cms_main(int argc, char **argv)
                 goto end;
 
             if (CMS_RecipientInfo_type(ri) == CMS_RECIPINFO_AGREE
-                && wrap_cipher) {
+                    && wrap_cipher != NULL) {
```
Comment (slontis) [inline apps/cms.c:973]: Will leave it up to you whether this change stays in this PR.

**Review by slontis — APPROVED** [review]

File: apps/cms.c (hunk 2 above)
Comment (t8m) [reply apps/cms.c:973]: IMO it is OK to stay.

**Review by t8m — APPROVED** [review]

File: apps/cms.c (hunk 2 above)
Comment (richsalz) [reply apps/cms.c:973]: yeah, it's a separate code-cleanup/style thing.   might as well leave it to avoid resetting the clock.  it would be done in #14182 anyway :)

**Comment by t8m** [comment]
Merged to master. Thank you for the contribution!
